MIP101/MIP101.md

```diff
@@ -1426,10 +1084,6 @@ Prime Delegates receive 60 million NewGovToken per year paid out through the AD
 
 Reserve Delegates receive 20 million NewGovToken per year paid out through the AD PDMs.
 
-###### 4.3.11.2.5: 20 million NewGovToken to Aligned Voter Committees
-
-Aligned Voter Committee Members with the highest number of verified NewGovTokens receive 20 million NewGovToken per year paid out through the AVC Module system.
-
```

Comment:
Not subtracted from the 200 million total for Governance upkeep emissions that are not recaptured in 4.3.11.2.